slash/core/session.py

```diff
@@ -51,6 +52,12 @@ def __init__(self, reporter=None, console_stream=None):
 
         self._skip_exc_types = (exceptions.SkipTest,)
 
+    def notify_internal_error(self, error): # pylint: disable=unused-argument
```

Comment:
Why should `error` be passed here, if it is not used? Is it for future use?

Reply:
Yes, although on second thought I'll drop it.